modern/src/Theme.js

```diff
@@ -0,0 +1,50 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const traccarPurple = '#333366';
+const traccarGreen = '#4CAF50';
+const traccarWhite = '#FFF';
+
+export default createMuiTheme({
+  palette: {  
+    common: {
+      purple: traccarPurple,
+      green: traccarGreen
+    },
+    primary: {
+      main: traccarPurple
+    },
+    secondary: {
+      main: traccarGreen,
+      contrastText: traccarWhite
+    }
+  },
+  overrides: {
+    MuiFormControl: {
+      root: {
+        height: '42px',
+      }
+    },    
+    MuiFilledInput: {
+      root: {
+        height: '42px',
+        borderRadius: '4px'
```

Comment:
We'll need to add a padding of 12px to the top to take care of the label when the input is focused or filled

Reply:
I meant margin. But I think we're better off without it. No need to implement